Kernel/Random.h

```diff
@@ -86,6 +86,16 @@ class FortunaPRNG {
         m_pools[pool].update(reinterpret_cast<const u8*>(&event_data), sizeof(T));
     }
 
+    bool seeded()
```

Comment:
This should be `const`. Also let's call it `is_seeded()` per the project coding style.